csquiz/index.html

```diff
@@ -1,81 +1,93 @@
-              Skip
-            </button><button id="button-reveal" class="btn btn-sm btn-outline-secondary me-2" type="button">
-              Show answer
-            </button><span id="incorrect" class="ms-4 text-danger text-nowrap d-none">
-              Incorrect! Try again.
-            </span>
+          <div id="button-next">
+            <button class="btn btn-primary" type="button">Next question</button>
           </div>
+        </div>
 
-          <div id="hint-alert" class="mt-3 pt-3 pb-0 alert alert-primary d-none">
-            <p><strong>Hint</strong></p>
-            <div id="hint-content"></div>
-            <br>
-          </div>
-        </form>
+        <div id="nav">
+          <button id="button-submit" class="btn btn-primary me-2" type="button">
+            Answer
+          </button><button id="button-hint" class="btn btn-sm btn-outline-secondary me-2" type="button">
+            Hint
+          </button><button id="button-skip" class="btn btn-sm btn-outline-secondary me-2" type="button">
+            Skip
+          </button><button id="button-reveal" class="btn btn-sm btn-outline-secondary me-2" type="button">
+            Show answer
+          </button><span id="incorrect" class="ms-4 text-danger text-nowrap d-none">
+            Incorrect! Try again.
+          </span>
+        </div>
+
+        <div id="hint-alert" class="mt-3 pt-3 pb-0 alert alert-primary d-none">
+          <p><strong>Hint</strong></p>
+          <div id="hint-content"></div>
+          <br>
+        </div>
+      </form>
 
-        <div id="footer" class="font-weight-light font-italic text-muted mt-3">
-          <p>You have answered
-            <span id="answered">0</span>
+      <div id="footer" class="font-weight-light font-italic text-muted mt-3">
+        <p>You have answered
+          <span id="answered">0</span>
           of
-            <span id="total">0</span>
-          questions correctly.</p>
+          <span id="total">0</span>
+          questions correctly.
+        </p>
 
-          <p><a href="what.html">What is this?</a></p>
-        </div>
+        <p><a href="what.html">What is this?</a></p>
       </div>
     </div>
+  </div>
+
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.min.js"
+    integrity="sha384-cuYeSxntonz0PPNlHhBs68uyIAVpIIOZZ5JqeqvYYIcEL727kskC66kF92t6Xl2V"
+    crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.7.0/highlight.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.7.0/languages/csharp.min.js"></script>
+  <script>(() => {
+      if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+        document.head.insertAdjacentHTML('beforeend', '<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.7.0/styles/atom-one-dark.min.css" />');
+      }
+      hljs.highlightAll();
+    })();</script>
```

Comment:
Self-invoking function to run after everything is loaded ready.

Matches the css selector that is driven by Browser/OS theme selection